app-website-4/ProductsApi/Controllers/ProductsController.cs

```diff
@@ -16,6 +17,7 @@ public ProductsController(ProductStore productStore)
             _mProductStore = productStore;
         }
 
+        [Authorize]
```

Comment:
I always prefer making everything authorized by default, and specifying `AllowAnonymous` on routes that are suitable.

You can implement that by following this: https://stackoverflow.com/a/34994917/5235388

Reply:
👍